dev/com.ibm.ws.microprofile.config.1.2_fat_tck/fat/src/org/eclipse/microprofile/config12/tck/ConfigTckPackageTest.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.microprofile.config12.tck;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import componenttest.annotation.AllowedFFDC;
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.MvnUtils;
+
+/**
+ * This is a test class that runs a whole Maven TCK as one test FAT test.
+ * There is a detailed output on specific
+ */
+@RunWith(FATRunner.class)
+public class ConfigTckPackageTest {
+
+    @Server("FATServer")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        server.startServer();
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        server.stopServer("CWMCG0007E", "CWMCG0014E", "CWMCG0015E", "CWMCG5003E", "CWWKZ0002E");
+    }
+
+    @Test
+    @AllowedFFDC // The tested deployment exceptions cause FFDC so we have to allow for this.
+    public void testConfigTck() throws Exception {
```

Comment:
This is the name of the testcase, the passing of which indicates ALL tests in the TCK passed. IN this case you might think it would be better as testConfig12TCK but that is fairly clear from the package name. Also - when experienced there is only a 'standard' failure message, so little point in triaging (double clicking on) this test - so in the longer tyerm it would be better to go ythe other way and have a really common recognisable name that was not spec or version spercific like testTCKKickoff - chat tomorrow Tom.